### DISCUSSION TITLE
Clarification of behavior in HA, i.e. cluster-init, setup

### DISCUSSION THREAD

**Discussion by gmarcy** [k3s-io/k3s#6579] [Q&A] [answered]
I have setup a cluster with HA embedded DB following [these](https://docs.k3s.io/installation/ha-embedded) instructions.

Am I correct that once the control plane has been established that the information has been properly distributed and that the specific node that was the first one that all the others joined is no longer special in any way?  i.e. if I lose that node I still have full k3s capability?  I started to wonder in the case where I wanted to add a new worker later and what would I use as the node url to perform that join when the original main node was gone.

Thanks for any clarity folks can provide on how this would work.

**Comment by brandond** [comment, marked as answer]
Yes, that node is no longer special. You can join additional servers and agents to any of the servers, as long as you have quorum.

**Reply by gmarcy** [reply]
@brandond thanks!
